engine/lib/access.php

```diff
@@ -116,13 +116,12 @@ function get_access_array($user_guid = 0, $site_guid = 0, $flush = false) {
  * @return int default access id (see ACCESS defines in elgglib.php)
  */
 function get_default_access(ElggUser $user = null, array $input_params = array()) {
-	global $CONFIG;
 
 	// site default access
-	$default_access = $CONFIG->default_access;
+	$default_access = elgg_get_config('default_access');
 
 	// user default access if enabled
-	if ($CONFIG->allow_user_default_access) {
+	if (elgg_get_config('allow_user_default_access') {
```

Comment:
Missing closing parenthesis. It's recommended you use an IDE with syntax checking.